### PR TITLE
remove translations in controllers, helpers, notifications, presenters

### LINKED ISSUE #4040: Remove remaining translations - part 1 (rb files)
## Description
This is a follow-up to #3930 and #3957. There are a few more translations in the codebase that need to be removed. This issue should cover all translations in `rb` files (controllers, helpers, notifications, presenters etc.)

## Note
There are a few trickier translations and I'm not sure how to update those, so they won't be covered here:
- `app/models/case_contact.rb`
- `app/decorators/volunteer_decorator.rb`

### PR DESCRIPTION
**Pull request by metamoni**
### What github issue is this PR for, if any?
Resolves #4040

### What changed, and why?
Translations have been replaced with hardcoded English strings

### How will this affect user permissions?
- Volunteer permissions: N/A
- Supervisor permissions: N/A
- Admin permissions: N/A

### How is this tested? (please write tests!) 💖💪
Existing tests should cover most of these

### CODE REVIEW

**Review by littleforest — APPROVED** [review]
Thanks for catching all of these!

**Comment by littleforest** [comment]
@metamoni could you take a look at the failing specs? I think they are legitimate failures, not flaky specs.

**Comment by metamoni** [comment]
@littleforest I fixed the failing specs. Let me know if there's anything else missing ☺️

File: app/presenters/case_contact_presenter.rb [hunk 1]
```diff
@@ -13,8 +13,8 @@ def display_case_number(casa_case_id)
 
   def boolean_select_options
```
Comment (compwron) [inline app/presenters/case_contact_presenter.rb:14]: we could improve this entire realm of code but this is a good easy replacement

**Review by compwron — APPROVED** [review]

**Pull request merged**